src/sentry/stacktraces/processing.py

```diff
@@ -33,6 +33,9 @@ def __eq__(self, other: object) -> bool:
     def __ne__(self, other: object) -> bool:
         return self is not other
 
+    def get_frames(self) -> Sequence[dict[str, str]]:
```

Comment:
```suggestion
    def get_frames(self) -> Sequence[dict[str, str | int | bool]]:
```

Unless we stringify everything at some point - do we?

(I'm basing this on frames like this one:

```json
{
    "function": "c",
    "filename": "app:///_next/static/chunks/main-aadfe6c52ca3d609.js",
    "abs_path": "app:///_next/static/chunks/main-aadfe6c52ca3d609.js",
    "lineno": 1,
    "colno": 1379,
    "in_app": false
},
```
)